src/libraries/Native/Unix/tryrun.cmake

```diff
@@ -1,4 +1,5 @@
 set(TARGET_ARCH_NAME $ENV{TARGET_BUILD_ARCH})
+set(CROSS_ROOTFS $ENV{ROOTFS_DIR})
```

Comment:
isn't this already set by eng/common/cross/toolchain.cmake ?

Reply:
That does not seems to be set soon enough and build fails. `src/coreclr/tryrun.cmake` has same line.